src/jfem.jl

```diff
@@ -90,8 +90,15 @@ export dx, ds,dS,dP
 *(expr::Union{Expression,Argument,Constant,Form,Function}, expr2::Union{Expression,Argument,Constant,Form,Function}) = Expression(expr.pyobject[:__mul__](expr2.pyobject) )
 *(expr::Real, expr2::Union{Expression,Argument,Constant,Form,Function}) = Expression(expr2.pyobject[:__mul__](expr) )
 *(expr::Union{Expression,Argument,Constant,Form}, expr2::Real) = Expression(expr.pyobject[:__mul__](expr2) )
+
++(expr::Union{Expression,Argument,Constant,Form}, expr2::Real) = Expression(expr.pyobject[:__add__](expr2) )
++(expr::Real, expr2::Union{Expression,Argument,Constant,Form,Function}) = Expression(expr2.pyobject[:__add__](expr) )
```

Comment:
`Union{Expression,Argument,Constant,Form,Function}` this should probably have an alias